src/db-service.ts

```diff
@@ -0,0 +1,95 @@
+// global
+import { Item } from 'graasp';
+import { sql, DatabaseTransactionConnectionType as TrxHandler } from 'slonik';
+// local
+
+/**
+ * Database's first layer of abstraction for Categorys
+ */
+export class SearchService {
+  private static allColumns = sql.join(
+    [
+      'id',
+      'name',
+      'description',
+      'type',
+      'path',
+      'extra',
+      'settings',
+      'creator',
+      ['created_at', 'createdAt'],
+      ['updated_at', 'updatedAt'],
+    ].map((c) =>
+      !Array.isArray(c)
+        ? sql.identifier([c])
+        : sql.join(
+            c.map((cwa) => sql.identifier([cwa])),
+            sql` AS `,
+          ),
+    ),
+    sql`, `,
+  );
+
+  // return items contain keyword in title
```

Comment:
`containing?`

Reply:
You can use js doc instead, if you want to express the return value. https://jsdoc.app/
```
/**
* <parameters>
* @return items containing keyword in title
*/
```